engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableBlending.java

```diff
@@ -19,11 +19,16 @@
 import org.terasology.rendering.dag.StateChange;
 
 /**
- * Instances of this class enable OpenGL's blending, i.e. to render transparent objects or to make
- * a composite of different images.
+ * Enables blending in the OpenGL pipeline.
+ * Can be used to render transparent objects or to make a composite of different images.
  *
- * See also StateChange implementation SetBlendFunction to set the source and destination factors
- * used by the blending process.
+ * Note that SetBlendFunction can be used to set the source and destination factors used by the blending process.
+ *
+ * OpenGL Default: DisableBlending
+ * Terasology Default: DisableBlending
```

Comment:
The term "DisableBlending" is a reference to a specific state change class, not OpenGL's nor Terasology's defaults. Keeping your format I'd write:

```
* OpenGL default: blending disabled
* Terasology default: blending disabled
```

In practice I'd simply write: 

`* By default both OpenGL and Terasology keep blending disabled as it is a relatively expensive operation.`